app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,16 @@
+class CommentsController < ApplicationController
+  def index
+    @comments = Comment.all
+  end
+
+  def new
+    @comment = Comment.new 
+  end
```

Comment:
`new` can't do anything if it doesn't have a corresponding `create` controller action!